packages/react/src/patterns/sub-patterns/ContentGroup/ContentGroup.js

```diff
@@ -44,7 +44,9 @@ const ContentGroup = ({ children, heading, customClassName, cta }) => {
         {children}
       </div>
       {cta && (
-        <div className={`${prefix}--content-group__cta-row`}>
+        <div
+          data-autoid={`${stablePrefix}--content-group_cta}`}
```

Comment:
@kennylam should change here too right?

Reply:
@jeffchew Holy moly what is going on with me today?